library/Icingadb/Widget/ItemList/BaseCommentListItem.php

```diff
@@ -101,7 +102,11 @@ protected function assembleVisual(BaseHtmlElement $visual)
 
     protected function createTimestamp()
     {
-        return new TimeAgo($this->item->entry_time);
+        if ($this->item->expire_time != 0) {
```

Comment:
Isn't the `!= 0` check superfluous? It just doesn't look right to me, especially since it doesn't use `!==`.